Makefile

```diff
@@ -79,11 +79,13 @@ BUILD_TAG ?=
 HV_CFG_LOG = $(HV_OUT)/cfg.log
 VM_CONFIGS_DIR = $(T)/misc/config_tools
 ASL_COMPILER ?= $(shell which iasl)
+DEB_BOARD = $(notdir $(BOARD))
+DEB_SCENARIO = $(notdir $(SCENARIO))
 
 .PHONY: all hypervisor devicemodel tools life_mngr doc
 all: hypervisor devicemodel tools
 	@cat $(HV_CFG_LOG)
-	@python3 misc/packaging/gen_acrn_deb.py acrn_all $(ROOT_OUT) --version=$(FULL_VERSION) --board_name=$(BOARD) --scenario=$(SCENARIO)
+	@python3 misc/packaging/gen_acrn_deb.py acrn_all $(ROOT_OUT) --version=$(FULL_VERSION) --board_name=$(basename $(DEB_BOARD)) --scenario=$(basename $(DEB_SCENARIO))
```

Comment:
When `BOARD` or `SCENARIO` is a file, the board/scenario names shall be fetched from the XML files, not the file names, as there is no guarantee that the file names and board/scenario names are identical.

Actually we already have such logic in the hypervisor (see the macro `determine_config` in hypervisor/scripts/makefile/config.mk). As the hypervisor is the only consumer of the `BOARD`/`SCENARIO` parameters, the top-level Makefile didn't need to parse them itself previously. But now with the introduction of the all-in-one package, this Makefile has to parse those parameters now, which is what causes the problem here.

If we want to go along this way, we have to copy some of the logic from hypervisor/scripts/makefile/config.mk here.

Reply:
Fixed. Get BOARD and SCENARIO information from the file $(HV_CFG_LOG).

Reply:
Fine with me.